frontend/src/components/toolTip/ToolTip.style.ts

```diff
@@ -0,0 +1,106 @@
+import { BorderRadius } from '@/styles/borderRadius';
+import { Assets, ColorPrimary, GrayScale } from '@/styles/colors';
+import { Spacing } from '@/styles/spacing';
+import { Typography } from '@/styles/typography';
+import { css } from '@emotion/react';
+import type { TooltipDirectionType, TooltipType } from '@/types/tooltip.type';
+
+const tooltipPosition = {
+  Top: css`
+    top: -60px;
+    left: 50%;
+    transform: translateX(-50%);
+  `,
+  Bottom: css`
+    bottom: -50px;
+    left: 50%;
+    transform: translateX(-50%);
+  `,
+  Left: css`
+    top: 50%;
+    right: 40px;
+    transform: translateY(-50%);
+  `,
+  Right: css`
+    top: 50%;
+    left: 40px;
+    transform: translateY(-50%);
+  `,
+};
```

Comment:
이거 아래처럼 default offset 을 두고서, 해당 offset 별로 값을 넣을 수 있게 하는 건 어때요? css-in-js의 장점을 극대화 하는 거예요 ㅎㅎ
```suggestion
export const tooltipPosition = {
  Top: (offset = 60) => css`
    top: -${offset}px;
    left: 50%;
    transform: translateX(-50%);
  `,
  Bottom: (offset = 50) => css`
    bottom: -${offset}px;
    left: 50%;
    transform: translateX(-50%);
  `,
  Left: (offset = 40) => css`
    top: 50%;
    right: ${offset}px;
    transform: translateY(-50%);
  `,
  Right: (offset = 40) => css`
    top: 50%;
    left: ${offset}px;
    transform: translateY(-50%);
  `,
};
```

Reply:
<div css={tooltipPosition.Top(72)} />
<div css={tooltipPosition.Left(24)} />
<div css={tooltipPosition.Bottom()} /> // 기본값 사용 (50px)
이런 식으로도 쓸 수 있을 것 같아요~^^

Reply:
헉 객체 안에도 파라미터를 넣어서 함수처럼 쓸 수 있을거라고 생각도 못했어요... 많이 배워갑니다 ㄷㄷ